lang/en.json

```diff
@@ -37,7 +37,7 @@
                 "multiple": "check out with multiple addresses",
                 "or": "or"
             },
-            "button": "Checkout",
+            "button": "Check out",
```

Comment:
👍  this one is always confusing, but I agree it should technically be the verb, as opposed to the noun.